CHANGELOG.md

```diff
@@ -9,6 +9,17 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Use hint diagnostics in vscode for better rendering in the editor (PR: [#69403](https://github.com/dotnet/roslyn/pull/69403))
+* Tell the language server to open projects if no solution exists (PR: [#6062](https://github.com/dotnet/vscode-csharp/pull/6062))
+* Show toast when project loading fails (PR: [#6060](https://github.com/dotnet/vscode-csharp/pull/6060))
+* Fix misleading lsp server logs (PR: [#69378](https://github.com/dotnet/roslyn/pull/69378))
+* Add support for code lens enable/disable options to roslyn LSP (PR: [#6001](https://github.com/dotnet/vscode-csharp/pull/6001))
+* Onboard Localization pipeline (PR: [#5990](https://github.com/dotnet/vscode-csharp/pull/5990))
+* Enable loading translated strings from razor TS code (PR: [#5962](https://github.com/dotnet/vscode-csharp/pull/5962))
+* Update typescript and eslint plugin to remove build warning (PR: [6002](https://github.com/dotnet/vscode-csharp/pull/6002))
```

Comment:
Should we include things that only impact us in changelogs like this? Or put them in a different section?

Reply:
My intent with including them was to give a rough idea of other work that were added to the extension, even though not user facing. But can move it to a separate section if thats better.